test/e2e/e2e_suite_test.go

```diff
@@ -161,6 +171,62 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		Expect(err).NotTo(HaveOccurred())
 	}
 
+	// Metrics test start
+	By("creating a ClusterRoleBinding for the service account to allow access to metrics")
+	cmd := exec.Command("kubectl", "create", "clusterrolebinding", metricsRoleBindingName,
+		"--clusterrole=baremetal-operator-metrics-reader",
+		fmt.Sprintf("--serviceaccount=%s:%s", namespace, serviceAccountName),
+	)
+	_, err := utils.Run(cmd)
+	Expect(err).NotTo(HaveOccurred(), "Failed to create ClusterRoleBinding")
+
+	By("validating that the metrics service is available")
+	cmd = exec.Command("kubectl", "get", "service", metricsServiceName, "-n", namespace)
```

Comment:
Yes , I know but this is a direct adoption of controller-runtime's e2e test so ...

Reply:
currently yes, it took time to adapt it to our environment and since we want to add the RBACs before the release, this is the best I can do

Reply:
I will do a follow up if thats what you are after or you can do it as well , since this is in test-code, I didnt bother to reduce kubectl usage if thats what you are after

Reply:
Yes, my question is if it has to be copy/paste, or are you ok with improving it later on? as said I'm ok with having it like this, as long as we agree we will get rid of os.Exec later on.